core/primitives/src/account.rs

```diff
@@ -20,6 +20,17 @@ pub struct Account {
     pub storage_usage: StorageUsage,
 }
 
+impl Account {
+    pub fn new(amount: Balance, code_hash: CryptoHash) -> Self {
+        Account {
+            amount,
+            locked: 0,
+            code_hash,
+            storage_usage: std::mem::size_of::<Account>() as u64,
```

Comment:
This is not the default value we use. We use `100` storage usage right now for a new account, so we can't switch to mem size of the account structure.  This is the reason it was in the `test_utils`

Reply:
@evgenykuzyakov Good catch!

@fckt What is the reason we moved all the stuff from `test_utils.rs`? It is not great to have it there, but neither it is good to have those methods mixed with the production methods.

I vote for having those test methods hidden behind an ad-hoc `Testing` trait (use `easy-ext` for easy creation of the ad-hoc traits). In any case, we should not have these methods mixed with the production implementations.

Reply:
> @fckt I vote for having those test methods hidden behind an ad-hoc Testing trait (use easy-ext for easy creation of the ad-hoc traits). In any case, we should not have these methods mixed with the production implementations.

This was exactly a reason. But I didn't know that this cannot be used as a default constructor.

> We use 100 storage usage right now for a new account, so we can't switch to mem size of the account structure.

thanks, good to know

Reply:
https://github.com/nearprotocol/nearcore/pull/2511/#issuecomment-620906318